src/Tables/__tests__/Table.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { render } from 'test-utils';
 import convert from 'color-convert';
 import { strictEqual } from 'assert';
-import TableCell from '../TableCell';
+import TableCell, { ALIGN } from '../TableCell';
```

Comment:
I would add some tests in here to make sure that:

- `ALIGN.LEFT`, `ALIGN.CENTER` and `ALIGN.RIGHT` do what you expect
- The default value is `ALIGN.LEFT`

Reply:
Added tests through 5619e56